ui-v2/app/components/changeable-set/index.hbs

```diff
@@ -1,4 +1,5 @@
 {{yield}}
+  <YieldSlot @name="content" @params={{block-params items}}>{{yield}}</YieldSlot>
```

Comment:
Moving forwards the individual `Consul*Listing` will be responsible for displaying an empty state, not the `ChangeableSet`. We made a new block here so we can support both approaches at once.